sale_blanket_order/models/blanket_orders.py

```diff
@@ -181,16 +184,22 @@ class BlanketOrderLine(models.Model):
     sequence = fields.Integer()
     order_id = fields.Many2one(
         'sale.blanket.order', required=True, ondelete='cascade')
+    planned_line_ids = fields.One2many(
+        'sale.blanket.planned.order.line', 'blanket_order_line_id', string='Planned order lines')
     product_id = fields.Many2one(
         'product.product', string='Product', required=True)
     product_uom = fields.Many2one(
         'product.uom', string='Unit of Measure', required=True)
     price_unit = fields.Float(string='Price', required=True)
+    next_order_qty = fields.Float(string='next planned order Qty', compute='_compute_next_planned_order', store=True)
```

Comment:
please put the first letter in capital for all words in string attribute.

Reply:
Done!